### PR TITLE
Move alice & faber "agents" to aries-vcx crate

### PR DESCRIPTION
**Pull request by mirgee**
Signed-off-by: Miroslav Kovar <miroslavkovar@protonmail.com>

### CODE REVIEW

File: libvcx/src/api_lib/api_handle/disclosed_proof.rs [hunk 1]
```diff
@@ -137,6 +138,16 @@ pub fn send_proof(handle: u32, connection_handle: u32) -> VcxResult<u32> {
     })
 }
 
+pub fn send_proof_temp(handle: u32, connection: &Connection) -> VcxResult<u32> {
```
Comment (Patrik-Stas) [inline libvcx/src/api_lib/api_handle/disclosed_proof.rs:141]: forgot to remove maybe

**Comment by Patrik-Stas** [comment]
Awesome, let's remove the few unused  *_temp functions and get this merged

**Review by Patrik-Stas — DISMISSED** [review]

File: libvcx/src/api_lib/api_handle/disclosed_proof.rs (hunk 1 above)
Comment (mirgee) [reply libvcx/src/api_lib/api_handle/disclosed_proof.rs]: Good catch

File: libvcx/src/api_lib/api_handle/proof.rs [hunk 2]
```diff
@@ -110,6 +111,13 @@ pub fn send_proof_request(handle: u32, connection_handle: u32, comment: Option<S
     })
 }
 
+pub fn send_proof_request_temp(handle: u32, connection: &Connection, comment: Option<String>) -> VcxResult<u32> {
```
Comment (Patrik-Stas) [inline libvcx/src/api_lib/api_handle/proof.rs:114]: one more here ^_^

**Pull request merged**